be/src/olap/rowset/segment_v2/binary_dict_page.h

```diff
@@ -86,7 +86,7 @@ class BinaryDictPageBuilder : public PageBuilder {
         }
     };
     // query for dict item -> dict id
-    std::unordered_map<Slice, uint32_t, HashOfSlice> _dictionary;
+    phmap::flat_hash_map<Slice, uint32_t, HashOfSlice> _dictionary;
```

Comment:
It seems that the interface of flat_hash_map and the interface of the standard library are completely compatible?

Reply:
yes，so it is eazily to replace the standard library